example/Vagrantfile

```diff
@@ -0,0 +1,16 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "debian/buster64"
+  config.vm.provider "virtualbox" do |v|
+    v.memory = 1024
+    v.cpus = 2
+  end
+
+  # ~/.ssh/id_rsa.pub is a file in the host machine
+  config.vm.provision "file", source: "~/.ssh/id_rsa.pub", destination: "~/.ssh/me.pub"
```

Comment:
@felixminom I think we could avoid copying a public SSH key from the host to the VM if we use the private key generated by Vagrant when running `hap deploy`.

```
#  example/hap.yaml
ssh_args: "-i .vagrant/machines/default/virtualbox/private_key"
```